internal/rbd/nodeserver.go

```diff
@@ -149,6 +149,65 @@ func healerStageTransaction(ctx context.Context, cr *util.Credentials, volOps *r
 	return nil
 }
 
+// populateVolOptions update the fields in rbdVolume struct based on the request it received.
+func (ns *NodeServer) populateVolOptions(
```

Comment:
Can this function be re-used easier when it isn't a method of `NodeServer`? `ns` does not seem to be used in the function, so tying it to `NodeServer` is sub-optimal.

Reply:
Done. I was more inclined to keep it as  a method though.